operational/flexera/automation/generic_meta_parent/generic_meta_parent.pt

```diff
@@ -0,0 +1,1015 @@
+name "Generic Meta Parent Policy"
+rs_pt_ver 20180301
+type "policy"
+short_description "**NOTE: Meta policies are an alpha feature. Please consult the [README](https://github.com/flexera-public/policy_templates/blob/master/README_META_POLICIES.md) before use.** A generic meta parent policy that dynamically creates and manages child policies based on cost dimensions."
+long_description <<-EOS
```

Comment:
Since the long description isn't used anywhere, shouldn't this be left empty like other policy templates?

(Also kind of curious why this didn't trigger a Dangerfile alert)

Reply:
Fixed in https://github.com/flexera-public/policy_templates/pull/3396/commits/21059ab172e739d42ea0e80da156b82e8fe292a1